tests/Oryx.Integration.Tests/DotNetCore/DotNetCoreRuntimeVersion30Tests.cs

```diff
@@ -329,5 +329,49 @@ public async Task CanBuildAndRun_NetCore30WebApp_HavingExplicitAssemblyName()
                     Assert.Contains("Hello World!", data);
                 });
         }
+
+        [Fact]
+        public async Task CanBuildAndRun_NetCoreApp31WebApp()
+        {
+            // Arrange
+            var dotnetcoreVersion = "3.1";
```

Comment:
@kichalla do we want to test both stable and preview in this test?

Reply:
Stable wouldn't be available until December, so for now we can only test preview.